cluster/node-pools/worker-splitaz/stack.yaml

```diff
@@ -55,8 +55,8 @@ Resources:
       - Key: node.kubernetes.io/role
         PropagateAtLaunch: true
         Value: worker
-# only node pools without taints should be attached to Ingress Load balancer
-{{- if or (not (index $data.NodePool.ConfigItems "taints")) (eq (index $data.NodePool.ConfigItems "taints") "dedicated=skipper-ingress:NoSchedule") }}
+# only skipper-ingress node pools should be attached to Ingress Load balancer
+{{- if and (eq .Cluster.ConfigItems.skipper_attach_only_to_skipper_node_pool "true") (eq (index $data.NodePool.ConfigItems "taints") "dedicated=skipper-ingress:NoSchedule") }}
```

Comment:
I believe this must be `$data.Cluster` (or similar) because you're in a loop here.

Reply:
Also matching with the error message:
```
Fail to provision: failed to provision node pool worker-gpu: template: main/cluster/node-pools/worker-splitaz/stack.yaml:59:23: executing \"main/cluster/node-pools/worker-splitaz/stack.yaml\" at <.Cluster.ConfigItems.skipper_attach_only_to_skipper_node_pool>: can't evaluate field Cluster in type string, failed to provision node pool worker-node-tests: template: main/cluster/node-pools/worker-splitaz/stack.yaml:59:23:
```